pages/blog.js

```diff
@@ -1,14 +1,79 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import {
+	FacebookIcon,
+	FacebookShareButton,
+	TwitterIcon,
+	TwitterShareButton,
+} from 'react-share';
+
 export default function blog() {
 	return (
-		<div>
-			<h2>Blog</h2>
-			<p className='blog--p'>
-				Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam eget erat
-				luctus, condimentum lectus eu, egestas tortor. Donec posuere vestibulum
-				eros. Pellentesque elementum eleifend mollis. Fusce ac risus ut mauris
-				imperdiet dapibus. Fusce non libero nec nisl facilisis posuere. Donec
-				gravida ex at arcu bibendum, in porta quam viverra.
-			</p>
+		<div className='blog--wrapper'>
+			<div className='blog--items'>
+				<h2>Blog</h2>
+				<Image src='https://picsum.photos/200/300' height='100%' width='100%' />
+				<p className='blog--p'>Erick Askim</p>
+				<p className='blog--p'>-</p>
+				<p className='blog--p'>Aug 12,2021</p>
+				<p className='blog--p'>-</p>
+				<FacebookShareButton url='#'>
+					<FacebookIcon size={32} round={true} />
+				</FacebookShareButton>
+				<TwitterShareButton url='#'>
+					<TwitterIcon size={32} round={true} />
+				</TwitterShareButton>
+				<p className='blog--p'>Lorem ipsum</p>
+				<Link href='#'>→</Link>
+			</div>
+			<div className='blog--items'>
+				<h2>Blog</h2>
+				<Image src='https://picsum.photos/200/300' height='100%' width='100%' />
+				<p className='blog--p'>Erick Askim</p>
+				<p className='blog--p'>-</p>
+				<p className='blog--p'>Aug 11,2021</p>
+				<p className='blog--p'>-</p>
+				<FacebookShareButton url='#'>
+					<FacebookIcon size={32} round={true} />
+				</FacebookShareButton>
+				<TwitterShareButton url='#'>
+					<TwitterIcon size={32} round={true} />
+				</TwitterShareButton>
+				<p className='blog--p'>Lorem ipsum</p>
+				<Link href='#'>→</Link>
+			</div>
+			<div className='blog--items'>
+				<h2>Blog</h2>
+				<Image src='https://picsum.photos/200/300' height='100%' width='100%' />
+				<p className='blog--p'>Erick Askim</p>
+				<p className='blog--p'>-</p>
+				<p className='blog--p'>Aug 10,2021</p>
+				<p className='blog--p'>-</p>
+				<FacebookShareButton url='#'>
+					<FacebookIcon size={32} round={true} />
+				</FacebookShareButton>
+				<TwitterShareButton url='#'>
+					<TwitterIcon size={32} round={true} />
+				</TwitterShareButton>
+				<p className='blog--p'>Lorem ipsum</p>
+				<Link href='#'>→</Link>
```

Comment:
Links should have a meaningful name. You'll want to add `aria-label` or `aria-labelledby`. [See here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute)